code/modules/mentor_fulp/mentor_bindings_fulp.dm

```diff
@@ -6,11 +6,11 @@
 	return user.mentor_datum ? TRUE : FALSE
 
 /datum/keybinding/mentor/mentor_say
-	hotkey_keys = list("F4")
+	hotkey_keys = list("F4") //What the absolute fuck
 	name = "mentor_say"
 	full_name = "Mentor say"
 	description = "Talk with fellow mentors and admins."
 
 /datum/keybinding/mentor/mentor_say/down(client/user)
 	user.get_mentor_say()
-	return TRUE
+	return ..()
```

Comment:
Two things here. 
1: For some reason the mentor key is f4. This feels like it will end poorly.
I assume that's why
2: This line removes the parent call of the keybinding. I assume this is to prevent alt f4 memes, but just use a different key man.